pubspec.yaml

```diff
@@ -1,7 +1,7 @@
 name: qr
 version: 0.1.0+1
```

Comment:
change this to `0.1.1-dev` – since it's pre-release

Or flag it as 0.1.0+2 and add a changelog entry – worth publishing now!